docs/recipes/template.rst

```diff
@@ -0,0 +1,36 @@
+Recipe Title
```

Comment:
We should make sure that this doesn't end up in the actual website. We can still include it in the repository for people to use as a template, but I think the cookbook would be cleaner if we just have user-friendly things there.

Reply:
Yes, sorry, I assumed that the first person adding a recipe will substitute the template call with the first recipe call, so that the template won't make it to the final documentation.